cmd/kubernetes/kubernetes_list.go

```diff
@@ -52,15 +52,28 @@ If you wish to use a custom format, the available fields are:
 			ow.AppendDataWithLabel("id", cluster.ID, "ID")
 			ow.AppendDataWithLabel("name", cluster.Name, "Name")
 			ow.AppendDataWithLabel("cluster_type", cluster.ClusterType, "Cluster-Type")
-			ow.AppendDataWithLabel("region", client.Region, "Region")
 			ow.AppendDataWithLabel("nodes", strconv.Itoa(len(cluster.Instances)), "Nodes")
 			ow.AppendDataWithLabel("pools", strconv.Itoa(len(cluster.Pools)), "Pools")
-			ow.AppendDataWithLabel("status", utility.ColorStatus(cluster.Status), "Status")
+
+			if cluster.Conditions != nil {
+				conditions := ""
+				for _, condition := range cluster.Conditions {
+					if condition.Type == "ControlPlaneReady" {
```

Comment:
can you do the same here `if cond.Type == ControlPlaneReady` like in cmd/kubernetes/kubernetes_show.go

Reply:
or made the cons global and used everywhere?